### ISSUE TITLE
Invalid line number when newlines above issue

### ISSUE THREAD

**Issue by mensfeld** [presidentbeef/brakeman#999]
When above an issue there's a string with multiple new lines and a hash (doesn't matter if for interpolation or not) Brakeman will count those all newlines from the string add add them into the line 

**Rails: 5.0.1**
**brakeman 3.5.0**
**Reproduction**: just copy paste this model into models:

```ruby
class Action < ActiveRecord::Base
  def not_important_method
    "\n\n\n\n\n\n\n\n\n#"
  end

  def insecure_method
    update_all("#{possible_sql_injection}")
  end
end
```

and run:

```
brakeman ./ -f json
```

Output (line is 16 instead of 7):

```
  "warnings": [
    {
      "warning_type": "SQL Injection",
      "warning_code": 0,
      "fingerprint": "7e61c1b95b6af784fd7643fcda1ce3eedcd6a74dce4204927de7bcfc0a24cec9",
      "check_name": "SQL",
      "message": "Possible SQL injection",
      "file": "app/models/test.rb",
      "line": 16,
      "link": "http://brakemanscanner.org/docs/warning_types/sql_injection/",
      "code": "update_all(\"#{possible_sql_injection}\")",
      "render_path": null,
      "location": {
        "type": "method",
        "class": "Action",
        "method": "insecure_method"
      },
      "user_input": "possible_sql_injection",
      "confidence": "Medium"
    }
   ]
```

**Comment by presidentbeef** [comment]
Hi Maciej,

Thank you for reporting. That is pretty weird that only happens with `#` at the end of the string.

**Comment by mensfeld** [comment]
@presidentbeef  I know - but the case happens for both interpolated and not interpolated string ```"\n\n\n#"``` or ```"\n\n\n#{something}"```. Were you able to reproduce that or do I need to give you more details?

**Comment by presidentbeef** [comment]
I am able to reproduce it, thanks!

**Comment by mensfeld** [comment]
@presidentbeef  good luck :) seems like a crazy issue :D

**Comment by mensfeld** [comment]
Hey @presidentbeef  any news on that one? :-)

**Comment by presidentbeef** [comment]
Possibly fixed by https://github.com/seattlerb/ruby_parser/pull/247

**Comment by presidentbeef** [comment]
Fixed in Brakeman 3.7.0.
